lib/releases.sh

```diff
@@ -39,8 +39,8 @@ if [[ "${CAPM3_RELEASE_PREFIX}" =~ ^(1\.6|1\.7|1\.8|1\.9)$ ]]; then
   export CAPM3RELEASE="v${CAPM3_RELEASE_PREFIX}.99"
   CAPI_RELEASE_PREFIX="v${CAPM3_RELEASE_PREFIX}."
 else
-  export CAPM3RELEASE="v1.10.99"
-  CAPI_RELEASE_PREFIX="v1.9."
+  export CAPM3RELEASE="${CAPM3RELEASE:-"v1.10.99"}"
+  CAPI_RELEASE_PREFIX="${CAPI_RELEASE_PREFIX:-"v1.9."}"
```

Comment:
the version prefix mismatch between CAPM3 and CAPI would be very confusing though, although I know currently theres no other way

Reply:
v1.10.99 is the main branch of CAPM3, I think we should start testing CAPI main instead of latest(v1.9) with CAPM3 main. But let's discuss it further and do it in other PR.